tests/upgrades/test_subscription.py

```diff
@@ -189,7 +190,7 @@ def test_post_subscription_scenario_autoattach(self, request, dependent_scenario
             host=docker_vm,
         )[docker_vm]
         assert 'Subscribed' in subscription
-        ssh.command(
+        default_sat.execute(
             f"docker stop {docker_container_name}; docker rm {docker_container_name}",
             hostname=f'{docker_vm}',
```

Comment:
This command is supposed to execute against the docker vm host, not `default_sat`

Reply:
lol nice!